src/pages/Admin/Admin.tsx

```diff
@@ -91,7 +91,7 @@ const VenueList: React.FC<VenueListProps> = ({
         </Link>
       </div>
       <ul className="page-container-adminsidebar-venueslist">
-        {venues.map((venue, index) => (
+        {relatedVenues.map((venue, index) => (
           <li
             key={index}
```

Comment:
[nit] Since we're here, it's a trivial change, and I believe `relatedVenues` provides the `venue.id` for us, we should change this `key` to use the `id` rather than the `index`.

Reply:
done 06fc69c809796e1f2f134b311c6fe8f756ee5d4d